pkg/client/kubernetes/clientmap/internal/plant_clientmap.go

```diff
@@ -67,6 +67,7 @@ func (f *PlantClientSetFactory) NewClientSet(ctx context.Context, k clientmap.Cl
 		kubernetes.WithClientOptions(client.Options{
 			Scheme: kubernetes.PlantScheme,
 		}),
+		kubernetes.WithDisabledCachedClient(),
```

Comment:
Why are we doing this?

Reply:
We don't want to start caches for the nodes in `Plant`s. That's just unnecessary memory usage for a rather unused feature :D